### ISSUE TITLE
Feature Request: Highlight content equal to selection

### ISSUE THREAD

**Issue by chumbalum** [giuspen/cherrytree#470]
It would be nice to have a feature that highlights text which is equal to the selection. It's useful to quickly find same entries in a bigger text.

This is how it looks like in Notepad++

![grafik](https://user-images.githubusercontent.com/486034/53725420-4108da00-3e6c-11e9-92b6-7ebd4624a96d.png)

**Comment by philippHRO** [comment]
Similarly, I want to suggest highlighting search results in this way.

**Comment by giuspen** [comment]
I agree it would be useful but cannot consider before the porting to C++/gtk3

**Comment by deskobj** [comment]
Have to be careful with this, though, because depending on the implementation, on fairly big documents this may incur a great performance penalty. I remember I had to disable it at some point because it was making editing unbearable. (I think it was Notepad++ at the time, but they maybe made it more efficient now, because I didn't have such problems with my other installs and usages since.)
